features/events/syslog/pom.xml

```diff
@@ -260,5 +266,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <!-- HZN-742 -->
+    <dependency>
+	 <groupId>org.apache.curator</groupId>
+	 <artifactId>curator-test</artifactId>
+	 <version>2.0.0-incubating</version>
+	 <scope>test</scope>
+	</dependency>
+    
+    <dependency>
+	 <groupId>log4j</groupId>
+	 <artifactId>log4j</artifactId>
+	 <version>1.2.15</version>
```

Comment:
The log4j:log4j artifact should be removed. It's the old log4j 1.x artifact so you need to remove it and add exclusions to all artifacts that require log4j:log4j.

To see which dependencies require it, you can run 'mvn dependency:tree' and see which JARs have log4j:log4j as a child.